node/test/fixtures/relay-stats.js

```diff
@@ -0,0 +1,187 @@
+module.exports.verify = verify;
+
+function verify(assert, actual, expected) {
+    var keys = Object.keys(actual);
+    var i;
+    var key;
+    for (i = 0; i < keys.length; i++) {
+        key = keys[i];
+        if (typeof actual[key] === 'object') {
+            verify(assert, actual[key], expected[key]);
+        } else if (typeof expected[key] === 'function') {
+            expected[key](assert, actual[key]);
+        } else {
+            assert.equals(actual[key], expected[key]);
+        }
+    }
+}
+
+function isNumber(assert, value) {
+    assert.ok(typeof value === 'number', 'expected number');
+}
+
+function isLoHostPort(assert, value) {
+    var parts = value.split(':');
+    assert.ok(parts.length === 2, 'value ' + value + ' splits into two parts');
+    assert.ok(parts[0] === '127.0.0.1', value + ' is a lo host:port');
+    assert.ok(
+        parseInt(parts[1]).toString() === parts[1],
+        value + ' has number port'
+    );
+}
+
+var fixture = module.exports.fixture = [
```

Comment:
let's move this fixture array into the other test file like all the other request-stats tests etc.

You can keep the generic `verify` function here if you want.